dnf-docker-test/features/upgrade-2.feature

```diff
@@ -2,14 +2,18 @@ Feature: DNF/Behave test (upgrade test - ALL)
 
 Scenario: Install packages from repository "test-1"
  Given I use the repository "test-1"
- When I "install" a package "TestA, TestD, TestF, TestK" with "dnf"
- Then package "TestA, TestB, TestD, TestE, TestF, TestG, TestH, TestK, TestM" should be "installed"
- And package "TestC" should be "absent"
+ When I execute "dnf" command "dnf -y install TestA TestD TestF TestK" with "success"
+ Then transaction changes are as follows
+   | State        | Packages                                                       |
+   | installed    | TestA, TestB, TestD, TestE, TestF, TestG, TestH, TestK, TestM  |
+   | absent       | TestC                                                          |
 
 
 Scenario: Upgrade ALL from repository "upgrade_1"
  Given I use the repository "upgrade_1"
- When I execute "dnf" command "-y upgrade" with "success"
- Then package "TestA, TestB, TestD, TestE, TestF, TestG, TestH, TestM" should be "upgraded"
- And package "TestC" should be "absent"
- And package "TestK" should be "unupgraded"
+ When I execute "dnf" command "dnf -y upgrade" with "success"
+ Then transaction changes are as follows
+   | State        | Packages                                                |
+   | upgraded     | TestA, TestB, TestD, TestE, TestF, TestG, TestH, TestM  |
+   | absent       | TestC                                                   |
```

Comment:
what I would suggest for the future is to name packages based on their character. Probably rename `TestC` to `brokendeps` or so.